spec.bs

```diff
@@ -888,9 +890,9 @@ A <dfn for=fencedframetype>destination event</dfn> is either a
         1. Otherwise, set |attributionReportingEligibility| to "<code>[=eligibility/event-source=]</code>".
 
         1. Set |processResponse| to these steps given a [=response=] |response|:
-
+           1. Let |fenced| be true.
            1. Run [=process an attribution eligible response=] with |event|'s
-              [=destination enum event/attributionReportingContextOrigin=], |attributionReportingEligibility|, and |response|.
+              [=destination enum event/attributionReportingContextOrigin=], |attributionReportingEligibility|, |fenced|, |request|'s [=request/trigger verification metadata=], and |response|.
```

Comment:
The "process" algorithm accepts only a `trigger verification metadata`, but `[=request/trigger verification metadata=]` could be null. Can you handle this appropriately either by catching this and dipping out early, or asserting that it is never null on these kinds of requests perhaps? (It's not clear to me which is right).

Reply:
https://github.com/WICG/attribution-reporting-api/pull/1271 to address this in ARA spec, thanks.